src/main.rs

```diff
@@ -182,13 +189,20 @@ fn robots_txt() -> Option<content::RawText<&'static str>> {
 }
 
 #[get("/")]
-async fn index() -> Template {
-    render_index(ENGLISH.into()).await
+async fn index(
+    version_cache: &State<Arc<RwLock<RustVersion>>>,
```

Comment:
nit: can we use some kind of typedef for this?

Reply:
`trait Cached` + `type Cache<T> = ...` would probably be fine. Open to other opinions

Reply:
Yeah that's cleaner :+1: